internal/api/cluster_installation.go

```diff
@@ -490,17 +490,31 @@ func handleMigrateDNS(c *Context, w http.ResponseWriter, r *http.Request) {
 	// DNS Switch
 	clusterInstallationIDs := getClusterInstallationIDs(clusterInstallations)
 	newClusterInstallationIDs := getClusterInstallationIDs(newClusterInstallations)
-	c.Logger.Infof("Total DNS records to migrate: %s", len(clusterInstallations))
 	var installationIDs []string
+	var hibernatedInstallationIDs []string
```

Comment:
nit: these lines can be combined
```suggestion
	var installationIDs, hibernatedInstallationIDs []string
```